Training2/readme.md

```diff
@@ -300,7 +300,7 @@ parallel. SmartHLS leverages loop-level parallelism when pipelining a
 loop to allow multiple loop iterations to execute in parallel.
 Pipelining is a common hardware optimization to increase the throughput
 of a circuit that repeatedly processes data in a series of steps and
-achieve higher hardware utilization. We have already seen this kind of
+achieves higher hardware utilization. We have already seen this kind of
```

Comment:
Should be "achieve", since the clause "achieve higher hardware utilization" refers to pipelining, not the circuit. I understand the confusion, though, so we can add an Oxford comma after "and".